.eslintrc

```diff
@@ -5,7 +5,7 @@
     "ecmaFeatures": {
       "impliedStrict": true,
       "jsx": true,
-      "experimentalObjectRestSpread": true,
+      "experimentalObjectRestSpread": true
```

Comment:
ecmaFeatures.experimentalObjectRestSpread is deprecated https://eslint.org/docs/user-guide/configuring